apps/apps.humanatlas.io/src/app/pages/us6/embed-sidenav-content/embed-sidenav-content.component.html

```diff
@@ -32,6 +32,14 @@
       Documentation
     </button>
     <div class="filler"></div>
-    <button mat-flat-button hraPrimaryButton hraButtonSize="medium" [cdkCopyToClipboard]="code()">Copy</button>
+    <button
+      mat-flat-button
+      hraPrimaryButton
+      hraButtonSize="medium"
+      (click)="openSnackbar()"
```

Comment:
The snackbar should be opened on the `cdkCopyToClipboardCopied` event if the copying is successful.
https://material.angular.io/cdk/clipboard/api#CdkCopyToClipboard